README.md

```diff
@@ -1,2 +1,100 @@
 # scie-pants
-Protects your Pants from the elements.
+
+The scie-pants binary is intended to be the next-generation `./pants` script.
+
+It's currently in a phase of development suitable for trying out in any existing Pants-using
+project. We welcome early adopters who want to try it out and report successes, failures and where
+things work but could be better.
+
+## Installing
+
+For now, you'll need to download the correct binary for your system, mark it as executable and place
+it on your $PATH somewhere.
+
+The binaries are released via [GitHub Releases](https://github.com/pantsbuild/scie-pants/releases)
+for both Linux and macOS and both aarch64 and x86_64 chip architectures. Pants itself does not
+support Linux aarch64 quite yet, but `scie-pants` is ready for it!
+
+I run on Linux x86_64; so I install a stable release like so:
+```
+curl -fLO \
+  https://github.com/pantsbuild/scie-pants/releases/download/v0.1.9/scie-pants-linux-x86_64
+curl -fL \
+  https://github.com/pantsbuild/scie-pants/releases/download/v0.1.9/scie-pants-linux-x86_64.sha256 \
+  | sha256sum -c -
+chmod +x scie-pants-linux-x86_64 && mv scie-pants-linux-x86_64 ~/bin/scie-pants
+```
+
+You can then run `scie-pants` anywhere you'd use `./pants` and other places besides. It should work
+just like your existing `./pants` script but with ~33% less latency launching Pants.
+
+If you'd like to build you own version, see the [contribution guide](CONTRIBUTING.md). There are
+build instructions there.
+
+## Features
+
+Beyond support for all known existing `./pants` script features, the `scie-pants` executable
+provides the following:
+
++ A hermetic Python interpreter appropriate for the current active Pants version:
+
+  You no longer need to have Python of the right version, or even any Python at all, installed on
+  your system to use Pants. If you use `scie-pants` in a repo with old Pants (< 2.5.0), it will
+  fetch a CPython 3.8 interpreter to use and a CPython 3.9 interpreter otherwise. These interpreters
+  are self-contained from the [Python Build Standalone](
+  https://python-build-standalone.readthedocs.io/en/latest/) project.
+
++ The ability to run Pants in a subdirectory of your project:
+
+  This is of limited utility since Pants internals don't support this well at the moment, but as
+  soon as they do, `scie-pants` will allow you to work in the style you prefer.
+
++ Built in ability to set up a new Pants project:
+
+  If you run `scie-pants` in a directory where Pants is not already set up, it will prompt you, and
+  you can let it set up the latest Pants stable version for your project.
+
+## Caveats
+
+The `scie-pants` binary will re-install versions of Pants you have already installed. The underlying
+[`scie`](https://github.com/a-scie/jump/blob/main/README.md) technology uses an `nce` cache
```

Comment:
First working google result for "nce cache" is https://www.moko.ru/moko/nce.html ... should this link to something else explaining the acronym?

Reply:
I mean, I already link to the authoritative source in the very same sentence ...

Reply:
Hm. That page does not define `nce`... is it just the suffix of `science`, and not supposed to mean anything else? It _looks_ like an acronym.

Reply:
It does. At base it's just a name for a cache dir explained here: https://github.com/a-scie/jump/blob/b5b81a611d482f572d5369731b70279f5579c597/README.md?plain=1#L66-L70

And if you read out the last words of the last sentence its also explained.

Reply:
You well know my disdain for words. I'm very happy to review doc edit PRs that make things make sense to you and others.